src/digest/digest.ts

```diff
@@ -10,7 +10,7 @@ export const flattenHashArray = (data: any) => {
   return Object.keys(flattenedData).map(k => {
     const obj: any = {};
     obj[k] = flattenedData[k];
-    return keccak256(JSON.stringify(obj)).toString("hex");
+    return keccak256(JSON.stringify(obj));
```

Comment:
i think this changes the contract right? might affect downstream applications

Reply:
keccak function already returns hexxed string, what would change ?